README.md

```diff
@@ -3,27 +3,171 @@
 Bartholomew is a simple CMS-like tool for hosting a website. It is compiled entirely
 to WebAssembly, and can run in any Spin-capable system.
 
-![Bartholomew screenshot](static/bartholomew-screenshot.png)
+The screenshot below shows how Bartholomew is implemented as the official Fermyon website.
 
-Check out [the docs](/content/docs)
+![Bartholomew screenshot](docs/static/image/bartholomew-screenshot.png)
+
+## Bartholomew documentation
+
+Check out [the docs](https://bartholomew.fermyon.dev/) and please raise any issues or ask any questions; we are here to help :)
 
 ## Getting Bartholomew
 
 The easiest way to start is to get the [Bartholomew site template](https://github.com/fermyon/bartholomew-site-template) and then download the [latest release of Bartholomew](https://github.com/fermyon/bartholomew/releases).
 
```

Comment:
I think the Bartholomew binary is bundled in the site template repository (although it is currently not the latest binary).  If downloading the binary approach is the recommended method, I think a step indicating where the binary release must be placed would be helpful.